docs/DeveloperGuide.md

```diff
@@ -1,34 +1,255 @@
+but not all files will be modified. The above steps explains which lists will be modified after the commands listed above are called.
+For all other commands, they also call the overwrite functions, but they do not modify the state of any of the lists `aliasList`,  `historyList`, `notesList`,  `dailyRouteList` and `favouritesList`. 
+Thus, the `aliasList.txt`, `routesHistoryList.txt`, `notesList.txt`, `dailyRouteList.txt`  and `favouritesList.txt` inside the created `data` folder remains unchanged. <br/>
+#### Design Consideration
+Alternative 1 (current choice): Saves the entire list of block aliases, visited routes, tagged notes, daily routes and favourite locations. <br/>
+Pros: Easy to implement. <br/>
+Cons: Only highly effective when limited to use of one user. <br/>
+
+### 3.2. Daily route planning feature
+#### Current Implementation
+The current implementation is facilitated by `DailyRoute` class, with the `AddDailyRouteCommand` and `ShowDailyRouteCommand` subclasses invoking methods that the `DailyRoute` class provides. </br>
+`AddDailyRouteCommand` and `ShowDailyRouteCommand` extend `Command` (superclass), where `AddDailyRouteCommand` implements the feature of adding the schedule of the day to the `DailyRoute` object and `ShowDailyRouteCommand` accesses the `DailyRoute` object to retrieve an ArrayList with the location schedule provided from the `AddDailyRouteCommand` and run the routing algorithm present in the `Router` object. <br />
+`DaySchedulePair` class is implemented to act as a pair between a day input String and schedule ArrayList.
+Additionally, they implement the following operations:
+
+`addDailyRoute(String ,ArrayList<String>)` — Maps the inputted day string to the inputted ArrayList of the schedule of the day in a hashmap . <br />
+`getDailyRoute(String)` — Returns the schedule of the day that is mapped to the inputted day. <br />
+
+These operations are exposed in the `DailyRoute` class  as `DailyRoute#addDailyRoute()` and `DailyRoute#getDailyRoute(String)`. <br />
+Given below is an example usage scenario and how the addDailyRoute mechanism behaves at each step. <br />
+Step 1. The user launches the application.<br />
+Step 2. The user executes `add daily route` command. UI will then prompt the user `SELECT ENTRY TO ADD:`  to input a day index. <br />
+Step 3. The UI then prompts the user to input the next block that is in the day's schedule.  <br /> The inputted location will be appended to an ArrayList. <br />
+Step 4. Repeat step 3 until the word `END` is input by the user. <br /> 
+Step 5. The inputted day, and the filled Arraylist from step 3 is then passed into the DailyRoute object<br /> This done using the addDailyRoute method from the DailyRoute class <br />
+Step 6. The day and filled Arraylist passed in step 5 is then saved in a hashmap that the DailyRoute object contains. <br /> 
+
+The following image shows the sequence diagram in which the add day command is implemented
+![img.png](addday.png)
```

Comment:
![image](https://user-images.githubusercontent.com/77423978/113077771-5d6ed080-9204-11eb-9fa8-19e8e27215a6.png)

Do remember to also include the colon `:` at the front of the class.

Sequence diagram's line should extend instead of just ending on another object?

Our team does not have an exact fix to the issue but, we coded from scratch instead of copy/pasting. (Hope it helps)